package.json

```diff
@@ -164,6 +165,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^1.15.0",
     "react-test-renderer": "^16.7.0",
+    "react-testing-library": "^7.0.0",
```

Comment:
Since you've updated psammead with the addition of a new testing library, can you please also increment it and add a note in the changelog?